### PR TITLE
#162163224 Setup project to use sass 

### PR DESCRIPTION
**Pull request by SokoPaulSokool**
### What does this PR do?
This Pull Request sets up the project to use sass.

### Description of Task to be completed?
This involved installing node-sass to enable the project to interpret sass.
All the css files were changed to scss

### How should this be manually tested?
Clone the repo
Check out the ch-setup-redux-162163220 branch
run npm install
run npm start to start the development server.
The project is able to run with scss files in it which shows it can compile scss.

### What are the relevant pivotal tracker stories?
#162163224

### CODE REVIEW

**Review by bibangamba — APPROVED** [review]

**Review by kafuuma — APPROVED** [review]

**Review by joshtrigger — APPROVED** [review]

**Review by Kasulejoseph — APPROVED** [review]

**Review by KicaRonaldOkello — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -21,3 +21,7 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+/package-lock.json
```
Comment (JaymesKat) [inline .gitignore]: @SokoPaulSokool Good work so far however the `package.json` file should not be excluded.

Comment (SokoPaulSokool) [reply .gitignore]: package-lock.json is always causing errors when I try to build the app. It has some babel package.

Comment (SokoPaulSokool) [reply .gitignore]: I have removed it

Comment (JaymesKat) [reply .gitignore]: @SokoPaulSokool Go ahead and pick one of the two lock files and delete the other.

**Review by JaymesKat — APPROVED** [review]
@SokoPaulSokool LGTM

**Pull request merged**